modal-pop-up.css

```diff
@@ -0,0 +1,183 @@
+.no-scroll {
+  overflow: hidden;
+}
+
+/* wrapper */
+.ven-overlay {
+  position: fixed;
+  overflow-y: scroll;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  padding: 77px 0;
+  background: rgba(193, 199, 208, 0.4);
+}
+
+[aria-hidden="true"] {
+  display: none;
+}
+
+[aria-hidden="false"] {
+  display: block;
+}
```

Comment:
- [ ] I would kindly suggest you stick to the Figma design as much as possible as this will give some confidence in your ability to replicate any given design by a client. Current in the mobile view of your pop up cards i notice the stack list are moving out of the card and it makes it look different from what we have in the Figma design.
- [ ] Also the buttons are not properly aligned, i would also advise you to remove the excess space at the bottom of the buttons

| Figma Design 👇 | Your Design |
| ------------- | ------------- |
| <img width="190" alt="image" src="https://github.com/Paulinelali/microverse-personal-portfolio/assets/78634841/799354ce-332b-4d4e-b9ab-bc350316b283">  | ![ay_LI](https://github.com/Paulinelali/microverse-personal-portfolio/assets/78634841/9fab8f0c-c20e-4555-a2bd-a91459ff96fc)
  |